microsoft-edge/devtools-guide-chromium/performance-monitor/performance-monitor-tool.md

```diff
@@ -20,17 +20,17 @@ The **Performance monitor** tool helps determine where performance problems come
 <!-- ====================================================================== -->
 ## Open the Performance monitor tool
 
-To open the Performance monitor:
+To open the **Performance monitor** tool:
 
 1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
 
-1. In DevTools, on the main toolbar, select the **Performance monitor** tab.  If that tab isn't visible, click the **More tabs** (![More tabs icon](./performance-monitor-tool-images/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![More Tools icon](./performance-monitor-tool-images/more-tools-icon-light-theme.png)) button.
+1. In DevTools, on the Activity Bar, select the **Performance monitor** tab.  If that tab isn't visible, click the **More Tools** (![More Tools icon](./performance-monitor-tool-images/more-tools-icon-light-theme.png)) button.
```

Comment:
```suggestion
1. In DevTools, on the Activity Bar, select the **Performance monitor** tab.  If that tab isn't visible, click the **More Tools** (![More Tools icon](./performance-monitor-tool-images/more-tools-icon-light-theme.png)) button:
```
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/performance-monitor/performance-monitor-tool?branch=pr-en-us-2938#open-the-performance-monitor-tool

Reply:
done